index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/38b6d4ab77.js" crossorigin="anonymous"></script>
+    <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+        integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+   
+   
+    <link rel="stylesheet" href="./css/styles.css" type="text/css" />
+    <link rel="stylesheet" href="./css/styleanim.css" type="text/css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <title>African Cultural Dance Conference</title>
+</head>
+<body>
+    <div class="firefly"></div>
+    <div class="firefly"></div>
+    <div class="firefly"></div>
+    <div class="firefly"></div>
+    <div class="firefly"></div>
+    <div class="firefly"></div>
+    <div class="firefly"></div>
+   <!-- Header section start from here -->
```

Comment:
- [ ] [REQUIRED] Good job so far! Please `style` your project using the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) provided by `Microverse`. Your project must be the same as [Behance's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), you must make your project `color`, `shape`, `layout`, and `fonts` as shown on [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Please correct them also to the `MOBILE` version 🙏🏼

### Your design - `MOBILE`

![mobile](https://user-images.githubusercontent.com/63915024/235016129-36477eac-3210-4c4d-9710-e4397e787ffb.gif)


### [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) - `MOBILE`

![image](https://user-images.githubusercontent.com/63915024/235016221-37d4ac1f-0c38-438b-b3d8-c5c130cc8a79.png)